siliconcompiler/utils.py

```diff
@@ -230,3 +231,8 @@ def format_fileset_type_table(indent=12):
         table += f"{indent}{filetype:<10}| {fileset:<11}| {ext}\n"
 
     return table
+
+def default_credentials_file():
+    cfg_file = os.path.join(Path.home(), '.sc', 'credentials')
```

Comment:
`Path` objects also overload the `/` operator, you could do:

`Path.home() / '.sc' / 'credentials'`

...Although personally, I think that `os.path.join` is a little more clear.

Reply:
I've seen that used before, but I find it somewhat confusing at first